dali/operators/reader/loader/video_loader.cc

```diff
@@ -328,6 +321,18 @@ VideoFile& VideoLoader::get_or_open_file(const std::string &filename) {
     int width = codecpar(stream)->width;
     int height = codecpar(stream)->height;
     auto codec_id = codecpar(stream)->codec_id;
+
+    if (width == 0 || height == 0) {
```

Comment:
We do the same as in https://github.com/NVIDIA/DALI/blob/main/dali/operators/reader/loader/video/frames_decoder.cc#L160.